plugins/org.eclipse.fordiac.ide.export/src/org/eclipse/fordiac/ide/export/utils/DelayedFiles.java

```diff
@@ -124,17 +103,16 @@ public Path write(final Path path, final CharSequence data) {
 	 */
 	public Iterable<StoredFiles> write(final boolean forceOverwrite) throws IOException {
 		final ArrayList<StoredFiles> ret = new ArrayList<>(storage.size());
-
 		for (final FileObject fo : storage) {
 			File o = null;
-			final File f = fo.getPath().toFile();
+			final File f = fo.path().toFile();
 			if (!forceOverwrite && f.exists()) {
-				o = Utils.createBakFile(f);
+				o = createBakFile(f);
 			}
-			if (fo.getPath().getParent() != null) {
-				Files.createDirectories(fo.getPath().getParent());
+			if (fo.path().getParent() != null) {
+				Files.createDirectories(fo.path().getParent());
 			}
-			Files.write(fo.getPath(), fo.getData(), StandardCharsets.UTF_8);
+			Files.write(fo.path(), Collections.singleton(fo.data()), StandardCharsets.UTF_8);
```

Comment:
In the comment you state that the file is not written but stored in memory. However here you write to disk. Somehow this confuses me a bit.

Reply:
Yes, this part of the documentation was wrong since the day I wrote it.